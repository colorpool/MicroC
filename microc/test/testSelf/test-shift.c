int main(){
    int n;
    n=20;
    print 'd' : n;
    print 'd' :n<<2;
    print 'd' : n;
    print 'd' :n>>1;
    print 'd' : n;

}