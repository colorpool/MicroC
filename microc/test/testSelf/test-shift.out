24 19 0 5 25 15 1 13 0 20 12 15 -1 13 11 22 15 -1 13 0 1 3 15 -1 13 13 11 0 2 36 12 22 15 -1 13 11 22 15 -1 13 0 1 4 15 -1 13 13 11 0 1 35 12 22 15 -1 13 11 22 21 1